src/components/TextField.vue

```diff
@@ -304,11 +318,45 @@ export default {
 
     &:focus,
     &:active {
-      @apply border-gray-500;
+      @apply transition duration-150 border-gray-500 shadow-light;
     }
 
     &:hover {
-      @apply border-gray-400;
+      @apply transition duration-150 border-gray-400;
+    }
+  }
+
+  &--dark {
+    .text-field__label {
+      @apply text-gray-300;
+    }
+
+    .text-field__input {
+      @apply
+        text-white
+        border-gray-600
+        bg-gray-600;
+
+      &::placeholder {
+        color: rgba(0, 0, 0, 0);
+      }
+
+      &:placeholder-shown {
+        @apply py-4;
+
+        + .text-field__label {
+          transform: scale(1, 1);
+        }
+      }
```

Comment:
```suggestion
```

Seems like unnecessary duped code? It felt like this was causing a bug for me locally too.

Reply:
You are correct, removed